STYLE.md

```diff
@@ -109,6 +109,11 @@ A few general notes on our error handling philosophy:
   that there is complete test coverage for the degraded case.** Additionally, the user should be
   aware of the degraded state minimally via an error log of level warn or greater and via the
   increment of a stat.
+* If you do need to log a non-fatal warning or error, you can unit-test it with EXPECT_LOG_CONTAINS or
+  EXPECT_NO_LOGS from test/test_common/logging.h. It's generally bad practice to test by depending on
```

Comment:
Could use a GH link here, https://blog.github.com/2013-01-31-relative-links-in-markup-files/